app/views/providers/merits_reports/show.html.erb

```diff
@@ -11,6 +13,7 @@
         read_only: true,
       ) %>
 
+  <%# shared with CYA, RandP and means report %>
```

Comment:
Are these comments long term or just for a potential refactoring?

Reply:
i kind of liked having them but have removed now 👍